### PR TITLE
Added missing logging statements to C# API as well as summaries to both APIs

### PR DESCRIPTION
**Pull request by tjhrulz**
Somehow these got overlooked.

### CODE REVIEW

File: API/RainmeterAPI.cs [hunk 1]
```diff
@@ -54,6 +54,12 @@ public API(IntPtr rm)
         [DllImport("Rainmeter.dll", CharSet = CharSet.Unicode, CallingConvention = CallingConvention.Cdecl)]
         private extern static int LSLog(LogType type, string unused, string message);
 
+        [DllImport("Rainmeter.dll", CharSet = CharSet.Unicode, CallingConvention = CallingConvention.Cdecl)]
+        private extern static int RmLog(IntPtr rm, LogType type, string message);
+
+        [DllImport("Rainmeter.dll", CharSet = CharSet.Unicode, CallingConvention = CallingConvention.Cdecl)]
+        private extern static int RmLogF(IntPtr rm, LogType type, string message);
```
Comment (howaldmg) [inline API/RainmeterAPI.cs]: I didn't see an implementation for RmLogF in master.  I just saw the RmLogF mentioned in an .h file.  Is some implementation missing?  Or is the source for that not in this repo?

Comment (tjhrulz) [reply API/RainmeterAPI.cs]: It is being imported from the main Rainmeter source. You can find the implementation here: https://github.com/rainmeter/rainmeter/blob/master/Library/Export.cpp#L134

**Comment by howaldmg** [comment]
+1

File: API/RainmeterAPI.cs (hunk 1 above)
Comment (howaldmg) [reply API/RainmeterAPI.cs]: LIBRARY_EXPORT void __stdcall RmLog(void* rm, int level, LPCWSTR message);

LIBRARY_EXPORT void __cdecl RmLogF(void* rm, int level, LPCWSTR format, ...);


It was a little strange for me to see both __cdelc and __stdcall mixed together.  The change that introduced this was 3 years ago.  I wonder why each use a different export.

Comment (brianferguson) [reply API/RainmeterAPI.cs]: @howaldmg For `__stdcall`, the callee cleans the stack, so any vararg functions use `__cdecl`. Although the MSVC compiler will implicitly do this for us, it's good practice to define the correct convention.

-Brian

**Pull request merged**